### PR TITLE
Blog post on microservices with JHipster and OAuth 2.0

### PR DESCRIPTION
**Pull request by mraible**

### CODE REVIEW

File: _source/_posts/2018-03-01-develop-microservices-jhipster-oauth.md [hunk 1]
```diff
@@ -0,0 +1,498 @@
...
+author: mraible
+description: "This post shows you how to use JHipster to create a microservices architecture based on Spring Boot, Spring Security, and OAuth."
+tags: [jhipster, microservices, spring-boot, spring-security, oidc, oauth]
+tweets:
+- "Learn how to use JHipster to create a microservices architecture based on Spring Boot, Spring Security, and OAuth → "
+- "Did you know @java_hipster makes it easy to build microservice applications with @springboot and @angular? You can even use its @oauth_2 support to add SSO! "
+- "Wanna get started with @java_hipster and a microservices architecture based on @springboot and @springsecurity? We have just the 🎫! We'll even show you how to deploy it to @heroku! "
+---
+
+JHipster is a development platform to generate, develop, and deploy Spring Boot + Angular web applications and Spring microservices. It supports using many types of authentication: JWT, session-based, and OAuth 2.0 / OIDC. In its 5.0 release, it'll also add React as a UI option. The 5.0 version is under active development and will likely be released in the next few months.
```
Comment (nbarbettini) [inline _source/_posts/2018-03-01-develop-microservices-jhipster-oauth.md]: We should spell out the first instance of OpenID Connect.

File: _source/_posts/2018-03-01-develop-microservices-jhipster-oauth.md [hunk 2]
```diff
@@ -0,0 +1,498 @@
...
+Open your browser and go to `http://localhost:8761`. Log in with `admin/admin`, and you should see a welcome page that shows the gateway has registered.
+
+{% img blog/microservices-jhipster-oauth/jhipster-registry-with-gateway.png alt:"JHipster Registry with Gateway registered" width:"800" %}{: .center-image }
+
+## Generate a Blog Microservice Application
+
+In `okta-jhipster-microservices-oauth-example`, create a `blog` directory, and run `jhipster`.
+
+```
+mkdir blog && cd blog && jhipster
```
Comment (nbarbettini) [inline _source/_posts/2018-03-01-develop-microservices-jhipster-oauth.md]: Note for the future: `cmd1 && cmd2` doesn't work on Windows. To support our Windows readers I put these on separate lines.

Comment (mraible) [reply _source/_posts/2018-03-01-develop-microservices-jhipster-oauth.md]: Do you want me to fix these now or just next time? If next, please approve so I can merge.

Comment (nbarbettini) [reply _source/_posts/2018-03-01-develop-microservices-jhipster-oauth.md]: Next. Sorry about that, I picked Comment instead of Approve by mistake. :(

**Pull request merged**